requirements.in

```
@@ -3,6 +3,9 @@
 # File: https://raw.githubusercontent.com/apache/airflow/constraints-2.8.2/constraints-3.11.txt
 --constraint ./constraints.txt
 
+# Package management
+pip-tools
```

Should we tag version here? 
```suggestion
pip-tools==7.4.1
```